### DISCUSSION TITLE
Error in 2a_OMOP_ETL tutorial

### DISCUSSION THREAD

**Discussion by sonali-tamhankar** [som-shahlab/femr#179] [Q&A]
Hello, several of us are trying to walk through the tutorials, and we were able to do the first one. In the second tutorial, we are getting an error on the etl_generic_omop line, resulting in warnings that zero lines were written. Part of the error message says: 

```[MainThread  ] [ERROR]  Failing on input/omop/observation/2.csv _ConceptTableConverter(prefix='observation', file_suffix='', concept_id_field=None, string_value_field='value_as_string', numeric_value_field='value_as_number', concept_id_value_field='value_as_concept_id', force_concept_id=None)```

Any idea what might be the issue here? I am also getting an error for 4.csv.

**Comment by EthanSteinberg** [comment]
This appears to be a bug in the synthetic data generation. I created an issue to track this. https://github.com/som-shahlab/femr/issues/182

**Comment by sonali-tamhankar** [comment]
Hello,

Thank you. We were able to fix this by changing lines 250 and 256 in
extractors/omop.py.

I am now stuck on tutorial 5, training CLMBR. I am running on Windows 11,
WSL Ubuntu. I was able to install cuda successfully, I have an NVIDIA
GeForce RTX4060. The laptop has 64GB memory. I am getting segmentation
fault when I try to train CLMBR.

Has anyone successfully run this code on WSL? Just want to check that it is
supposed to run in this environment, as I continue to debug.

Thanks,
Sonali.
